phoenix-queryserver/src/it/java/org/apache/phoenix/end2end/QueryServerBasicsIT.java

```diff
@@ -161,5 +184,19 @@ public void smokeTest() throws Exception {
         }
       }
     }
+    assertFalse(" metrics file should not be empty", checkIfTempFileIsEmpty());
```

Comment:
Can you add a test which actually verifies that some of the metrics we'd expect to be in this file are there?

This test would silently pass if garbage/errors were somehow written to this file.